### DISCUSSION TITLE
Regex validation custom message

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#5220] [Feature Requests]
When setting a regex validation rule, it would be useful to be able to provide a custom message in case the validation fails. Showing the original regex to the user will mean nothing to most, and just showing "invalid value" without a hint on how to fix it isn't ideal either 🤔

**Comment by natcl** [comment]
Please implement this, would be very useful !
Thanks !

**Reply by rijkvanzanten** [reply]
How do you think this should work ideally? Add it into the configuration object under regex?

```json
{
  "title": {
    "_regex": "/$)QF()*!R)($!)(*$!/",
    "_regex_message": "Hello World!"
  }
}
```

**Reply by natcl** [reply]
I think I was confused by the idea, what I was thinking was to have regex validation at the schema level when adding text and string fields:
<img width="849" alt="image" src="https://user-images.githubusercontent.com/77556/117849403-459e5980-b252-11eb-951c-1e8fa319a55b.png">

**Reply by natcl** [reply]
I think what I'm looking for is described here:
https://github.com/directus/directus/discussions/4390

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
